crates/chisel/bin/main.rs

```diff
@@ -205,7 +207,7 @@ async fn main_args(args: Chisel) -> eyre::Result<()> {
     }
 
     // Print welcome header
-    println!("Welcome to Chisel! Type `{}` to show available commands.", "!help".green());
+    sh_println!("Welcome to Chisel! Type `{}` to show available commands.", "!help")?;
```

Comment:
cc @DaniPopes in https://github.com/foundry-rs/foundry/pull/9109 you noted messages ought not to be painted in color except for the `warn / error` keyword, is removing this here correct?

Reply:
Yeah but that's for err and warn, and here it's in a REPL too, so it's fine